lib/core/test/unit/Cardano/Wallet/Api/Malformed.hs

```diff
@@ -847,12 +847,31 @@ instance Malformed (BodyParam (ApiSelectCoinsData ('Testnet pm))) where
     malformed = jsonValid ++ jsonInvalid
      where
          jsonInvalid = first BodyParam <$>
-            [ ("1020344", "Error in $: parsing Cardano.Wallet.Api.Types.ApiSelectCoinsData(ApiSelectCoinsData) failed, expected Object, but encountered Number")
-            , ("\"1020344\"", "Error in $: parsing Cardano.Wallet.Api.Types.ApiSelectCoinsData(ApiSelectCoinsData) failed, expected Object, but encountered String")
+            [ ("1020344", "Error in $: parsing DelegationAction failed, expected Object, but encountered Number")
+            , ("\"1020344\"", "Error in $: parsing DelegationAction failed, expected Object, but encountered String")
             , ("\"slot_number : \"random\"}", "trailing junk after valid JSON: endOfInput")
             , ("{\"payments : [], \"random\"}", msgJsonInvalid)
+            , ("join", "I couldn't understand the content of your message. If your message is intended to be in JSON format, please check that the JSON is valid.")
+            , ("quit", msgJsonInvalid)
+            ]
+         jsonValid = (first (BodyParam . Aeson.encode) <$> paymentCases) <> jsonValidAction
+         jsonValidAction = first (BodyParam . Aeson.encode) <$>
+            [ ( [aesonQQ| { "action": "join" }|]
+              , "Error in $: No valid parse"
```

Comment:
The error is not quite satisfactory here, can we come up with something more meaningful which would help user getting to a resolution?